contents/tutorials/filter-session-recordings.md

```diff
@@ -0,0 +1,82 @@
+showTitle: true
+featuredImage: ../images/tutorials/banners/filter-sessions.png
+featuredTutorial: false
+author: ['joe-martin']
+topics: ['session recording']
+date: 2022-11-03
+---
+
+_Estimated reading time: 10 minutes_ ☕☕
+
+Session recordings can be a powerful feature, helping you to find hidden patterns in user behavior or friction points that you can’t isolate with [other insight types](/manual/insights). Funnels may tell you _where_ users are experiencing friction within your product, but only session recordings can show you _why_. 
+
+The challenge, however, can be finding relevant recordings and focusing on the right actions quickly. If you can’t do this, you can face the unenviable task for watching hundreds of recordings in their entirety. 
+
+In this tutorial we’ll explain how to use PostHog’s various filters and features to find relevant session recordings quickly, and painlessly. 
+
+## How to filter recordings by user behavior
+
+### How to filter recordings by actions or events
+
+The most typical use case for session recordings is to filter based on an event or action, so you can see how users interact with certain features or areas of your product. To do this, you need to have an appropriate event or action set up. 
+
+> Want to know more about creating actions and tracking events? Check [our complete guide to event tracking](/tutorials/event-tracking-guide). 
+
+To start filtering, simply select ‘Filter recordings’ at the top of the page. 
+
+![search for recording events](../images/tutorials/session-recordings/filter-events-session-recording.png)
+
+Searching for actions or events in PostHog is similar to building a query in PostHog insights — all you have to do is use the filter builder on the left of the screen to define what sort of actions or events you’re looking for. 
+Once you have filtered the recordings to a smaller, more relevant list then you can search within individual recordings to find the specific moment when an event occurs. In fact, there are a few ways to find the specific event you’re looking for!
+
+![global filter off](../images/tutorials/session-recordings/recording-filter-off.png)
+
+The easiest way to focus in on a particular action that you’ve created a filter for is to use the toggle labeled ‘Only show matching events’. This will remove all events which don’t match your global filter from both the timeline and the event list, changing the view from the above, to the below. 
+
+![global filter on](../images/tutorials/session-recordings/recording-filter-on.png)
+
+Alternatively, you may prefer to search for an event which _doesn’t_ match a global filter — in which case you use the search events field to find other events within the current recording. 
+
+### How to filter based by feature flags
+
+If you use feature flags in PostHog then you can also filter recordings by the flag variant shown to a user. This can be helpful if, for example, you want to compare user behavior between flag variants. 
```

Comment:
```suggestion
If you use feature flags in PostHog, you can also filter recordings by the flag variant shown to a user. This can be helpful if, for example, you want to compare user behavior between flag variants. 
```